tools/__init__.py

```diff
@@ -4,3 +4,9 @@
 ASAPP_ROOT = config.env_vars['ASAPP_ROOT']
 ASAPP_PRODML_ROOT = config.env_vars['ASAPP_PRODML_ROOT']
 ASAPP_MLENG_ROOT = config.env_vars['ASAPP_MLENG_ROOT']
+
+
+CLIENT_FULL_NAMES = {
+    'condor': 'comcast',
+    'spear': 'sprint'
+}
```

Comment:
again with this, I don't think constants should be in the __init__.py file. 

I'm fine with using it either of these ways:
`import constants; some_func(constants.AWFUL)`
`from constants import *; some_func(AWFUL)`

https://stackoverflow.com/questions/6345840/whats-the-best-way-to-initialise-and-use-contants-across-python-classes

Reply:
Changed.